### DISCUSSION TITLE
Help with getting wider post content

### DISCUSSION THREAD

**Discussion by m1evankaiser** [mmistakes/minimal-mistakes#1373] [Q&A] [answered]
- [x] This is a question about using the theme.
- [ ] This is a feature request.
- [ ] I believe this to be a bug with the theme.
  - [ ] I have updated all gems with `bundle update`.
  - [ ] I have tested locally with `bundle exec jekyll build`.

---

## Environment informations

- **Minimal Mistakes version:** 4.6.0
- **Jekyll version:** 3.6.2
- **GitHub Pages hosted:** yes
- **Operating system:** MacOS Sierra

---

## Expected behavior

I'd like to know how to adjust the width of the content such that the code blocks seen on [this page](http://devblog.members1st.org/dotnet-core-rest-api-in-about-30-minutes/) show more content without scrolling.

Ironically, if I shrink the width of the browser window until it gets to the large and medium breakpoints, you can see a lot more of the text without scrolling. However, the x-large view is both a narrower column of text and bigger font, resulting in a hard to read post. 

I found in `_variables.scss`, that it has a series of `$right-sidebar-width` variables, however, lowering those, which did help increase the content width, also seemed to negatively affect the left-sidebar (making it more squished) and I don't want to affect the left-sidebar, as it is fine the way it is.

## Steps to reproduce the behavior

Example page: http://devblog.members1st.org/dotnet-core-rest-api-in-about-30-minutes/
Repo: https://github.com/Members1stFCU/Members1stFCU.github.io

Steps to reproduce:
- Cloned the theme
- Customized `_config.yml`, `_data/navigation.yml`, and some other non-style related changes
- Added some posts
- Tested locally
- Pushed up to GitHub Pages

**Comment by mmistakes** [comment, marked as answer]
Have a search through the closed issues, I've covered this one a few times before.

You need to alter the CSS some. If you look through `_sass/_page.scss` you'll see I'm applying some negative margins on the right. That's "faking" some space to make sure an optional right sidebar fits. If you want the main content to expand into that space, removing the margins will do it.

```scss
.page {
  width: calc(100% - #{$right-sidebar-width-narrow});
}
```

**Reply by Rmorais1974** [reply]
Hello can you help to solve this issue I don't know where is this /_page.css

**Comment by mmistakes** [comment]
See https://github.com/mmistakes/minimal-mistakes/issues/1265

**Comment by m1evankaiser** [comment]
Thanks that is much better! I saw a mention about removing the sidebar padding, but thought that meant changing the sidebar width values in _variables.scss.

**Comment by amosshi** [comment]
FYI: I followed the guide in [this page](https://stackoverflow.com/questions/55765988/how-can-i-get-rid-of-the-max-width-1280-without-using-the-wide-class-in-minimal) and it worked perfectly:

Add property bellow to the main.scss file:
* `$max-width:90%;`

Of course, we can specify any other value for the `max-width` .

See #2093
